src/map.cpp

```diff
@@ -415,21 +412,30 @@ void Map::unspreadLight(enum LightBank bank,
 */
 void Map::unLightNeighbors(enum LightBank bank,
 		v3s16 pos, u8 lightwas,
-		std::set<v3s16> & light_sources,
+		std::vector<v3s16> & light_sources,
 		std::map<v3s16, MapBlock*>  & modified_blocks)
 {
-	std::map<v3s16, u8> from_nodes;
-	from_nodes[pos] = lightwas;
+	std::vector<std::pair<v3s16, u8> > from_nodes;
+	from_nodes.push_back(std::make_pair(pos,lightwas));
 
 	unspreadLight(bank, from_nodes, light_sources, modified_blocks);
 }
 
+namespace {
```

Comment:
why a namespace for this? Couldn't this function be usefull for other things too and be placed in a more generic place?